### PR TITLE
Mod downloader page improvements

### LINKED ISSUE #1465: Indicate whether a Modrinth project is archived when being installed
### Role

I maintain Minecraft modpacks and play modded Minecraft.

### Suggestion

I would like if the launcher indicated that a project is archived.

### Benefit

So that users know that a project they are installing is archived and will not receive updates.

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

### You may use the editor below to elaborate further.

Currently, if you are trying to install a project that is archived, there is no clear indication that it is archived. Modrinth has a feature to mark a project as archived to tell the user that it will no longer receive updates. 

I recently archived two of my modpacks: [Hammer](https://modrinth.com/modpack/hammer) and [Vulkadrenaline](https://modrinth.com/modpack/vulkadrenaline), which on the site clearly indicates that they are discontinued, but in the launcher it is not clear to the user.

###### on the website
modrinth shows that my project was archived above the description
modrinth shows that my project was archived through the tags area

###### in the launcher
launcher does not show that my project was archived

As a workaround, I must put this in my summary to indicate that it is no longer recommended to be used.

summary that tells the user that my project was archived and is not recommended for use

This would be a very useful feature as I don't really want people trying to install my archived projects, but I do want to allow other people to still see the source of the project and install it if they *really* want to.

Here's a crappy mockup I made:

launcher shows that the project is archived by displaying "archived" next to the title of the project
launcher contains a note in the description of the project that shows it is archived

### LINKED ISSUE #1864: Lines in mod summaries occasionally go over other mods
### Operating System

_No response_

### Version of Prism Launcher

Prism Launcher 8.0

### Version of Qt

Qt 6.6.0

### Description of bug

In the mod download menu, multiline summaries usually go under the next mod. However, if the mod right below one is batch loaded after the other, the summary will go over the mod below.

image

### Steps to reproduce

Due to the constant influx of new mods, these specific conditions are difficult to reproduce. The bug notably doesn't happen if the mods aren't separated by batch loads.

### Suspected cause

It'd probably be wise to trunctate everything after a newline in the summary, as the UI isn't designed around holding more than one line in the summary anyways.

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

### PR DESCRIPTION
**Pull request by lumiscosity**
Fixes #1465 
Fixes #1864 

Signed-off-by: lumiscosity <averyrudelphe@gmail.com>

### CODE REVIEW

File: launcher/ui/pages/modplatform/ResourceModel.cpp [hunk 1]
```diff
@@ -77,7 +77,7 @@ auto ResourceModel::data(const QModelIndex& index, int role) const -> QVariant
         case UserDataTypes::TITLE:
             return pack->name;
         case UserDataTypes::DESCRIPTION:
-            return pack->description;
+            return pack->description.simplified();
```
Comment (Trial97) [inline launcher/ui/pages/modplatform/ResourceModel.cpp]: I would prefer that the simplified call be made in ProjectItem.cpp on "Description painting" part removing the need to modify the ResourceModel and  ModrinthModel

Comment (lumiscosity) [reply launcher/ui/pages/modplatform/ResourceModel.cpp]: understandable, pushing a fix now

**Review by Trial97 — APPROVED** [review]

**Review by TheKodeToad — APPROVED** [review]

**Comment by Trial97** [comment]
Please run clang format on your code(or fix the styling manually)

**Comment by lumiscosity** [comment]
styling should be fine now

**Pull request merged**